packages/helper/src/slack/types/IDonationSlack.ts

```diff
@@ -2,8 +2,8 @@ import { IWishCard, IUser } from '@donategifts/common';
 
 export interface IDonationSlack {
 	service: string;
-	userDonation: string;
+	userDonation: number | null;
```

Comment:
same here, can it be null or undefined? if undefined you can write it like so ```userDonation?: number;```

Reply:
You could do it here, but I would rather stay consistent (and keep types the same, otherwise we will have to cast to undefined or number here).